types/reactstrap/lib/Util.d.ts

```diff
@@ -0,0 +1,3 @@
+import { CSSModule } from '../index';
```

Comment:
Can you please name this file utils.d.ts instead to match the lib structure in the package.